blackbox/docs/admin/system-information.rst

```diff
@@ -1421,7 +1421,7 @@ Here's an example query::
   |  1 | The setting 'discovery.zen.minimum_master_nodes' must not be ... |
   |  2 | The total number of partitions of one or more partitioned tab... |
   |  3 | The following tables need to be upgraded for compatibility wi... |
-  |  6 | Your CrateDB license is not close to expiry. Enjoy CrateDB!      |
+  |  6 | Your CrateDB license is fine. Enjoy CrateDB!                     |
```

Comment:
could we change 'fine' to 'valid' or may be rephrase it to something like "You have a valid CrateDB license. Enjoy CrateDB"?